config/features.yml

```diff
@@ -226,6 +226,10 @@ features:
     actor_type: user
     description: Uses local_bgs rather than bgs-ext
     enable_in_development: true
+  claims_api_find_by_ssn_uses_person_web_service:
```

Comment:
Just wanted to call out that this is using the same feature flag as [this PR](https://github.com/department-of-veterans-affairs/vets-api/pull/19512). Assuming that's intentional?